lib/item_repository.rb

```diff
@@ -0,0 +1,39 @@
+require_relative './item'
+require './app'
+
+class ItemRepository
+    def all
+      sql = 'SELECT * FROM items;'
+      result = DatabaseConnection.exec_params(sql, [])
+      items = []
+      result.each do |record|
+        item = Item.new
+        item.id = record['id']
+        item.name = record['name']
+        item.price = record['price']
+        item.quantity = record['quantity']
+        items.push(item)
+      end
+    return items
+    end
+    
+    def find(id)
+        sql = 'SELECT id, name, price, quantity FROM items WHERE id = $1'
+        sql_params = [id]
+        result = DatabaseConnection.exec_params(sql, sql_params)
+        record = result[0]
+        item = Item.new
+        item.id = record['id']
+        item.name = record['name']
+        item.price = record['price']
+        item.quantity = record['quantity']
+        return item
+    end
+
```

Comment:
clear spacing throughout the file so that each method in the class is easy to read